src/js/shariff.js

```diff
@@ -112,6 +112,12 @@ Shariff.prototype = {
 
         flattrUser: null,
 
+        createLinkIcon: function($shareLink, service) {
+            if (typeof service.faName !== 'undefined') {
+                $shareLink.prepend('<span class="fa ' +  service.faName + '">');
```

Comment:
I copied the code from the source. But your right, I can change it if you would like ;)